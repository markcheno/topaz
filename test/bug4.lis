#    0: adjs		0
#    5: jmp		115
#   10: link		1
#   15: pushint		3
#   20: newarray
#   21: dup
#   22: pushint		0
#   27: pushint		1
#   32: setarray
#   33: dup
#   34: pushint		1
#   39: pushint		2
#   44: setarray
#   45: dup
#   46: pushint		2
#   51: pushint		3
#   56: setarray
#   57: setlocal	0
#   62: getlocal	0
#   67: pushint		2
#   72: gtr
#   73: jz		110
#   78: pushint		0
#   83: pushstring	'a > b'
#   94: pushint		1
#   99: call		println
#  104: pop
#  105: jmp		110
#  110: rts		2
#  115: pushint		0
#  120: pushint		0
#  125: jsr		10
#  130: pop
#  131: halt
