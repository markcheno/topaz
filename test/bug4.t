

def xx

  a = [1,2,3]
  if a > 2 then
    println("a > b")
  end
end

xx
