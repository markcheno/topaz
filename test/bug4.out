runtime error - invalid type for: op_gtr
